dcpy/connectors/edm/recipes.py

```diff
@@ -159,7 +160,8 @@ def import_dataset(
         }
         df.rename(columns=columns, inplace=True)
         pg_client.insert_dataframe(df, ds_table_name)
-        pg_client.add_pk(ds_table_name)
+        if id_column is None:
+            pg_client.add_pk(ds_table_name)
```

Comment:
Should maybe do this differently. This is letting you specify what column in the input dataset is the id column. In the case of this dataset, there is a column called `id` that seems to be serial. Since the dataset has an id column, none is added.

However, this column isn't then specified as a primary key/unique index. So I could add that. However, that might be making assumptions about data and validation in a spot that we don't really want it (at time of import). And in that case, maybe here it would make more sense to specify a different label for a newly-generated id column rather than the label for the existing one in the dataset.

Thoughts?

Reply:
hmmm yea I feel like since the only effect of specifying `id_column` is that a PK isn't generated we should just generate a new PK every time

that way, any builds that expect it will have it and there'd be no risk of being mislead by specifying a column that hasn't been checked. in the future when we refactor build logic we can always investigate, test, and use a source ID column and rely less of an expected `ogc_fid`-ish column generated at archival

Reply:
noting we're gonna try always adding `ogc_fid` on import when the source dataset isn't a pgdump